src/javascript/app/ng-wig/ng-wig-toolbar.provider.spec.js

```diff
@@ -5,126 +5,128 @@ describe('provider: ngWigToolbar', function () {
         module('ngWig', (_ngWigToolbarProvider_) => {
             toolbarProvider = _ngWigToolbarProvider_;
         });
+
+        inject();
```

Comment:
We need it so that the injector will be initialized and the ngWig module will be loaded